chia/util/initial-config.yaml

```diff
@@ -550,3 +546,27 @@ wallet:
 
   # Interval to resend unconfirmed transactions, even if previously accepted into Mempool
   tx_resend_timeout_secs: 1800
+
+simulator:
+
+  selected_network: testnet0
+
+  # The Introducer is set to connect locally because the simulator is not on Mainnet
```

Comment:
these variables are already present in full node config. We should remove it from here.

Reply:
Im removing selected_network and introducer, is that good?